safehttp/plugins/cors/cors.go

```diff
@@ -42,12 +43,13 @@ var disallowedContentTypes = map[string]bool{
 //
 // Each CORS request must contain the header "X-Cors: 1".
 //
+// The HEAD request method is disallowed.
+//
 // All of this is to prevent XSRF.
 type Interceptor struct {
 	// AllowedOrigins determines which origins should be allowed in the
 	// Access-Control-Allow-Origin header.
 	AllowedOrigins map[string]bool
-	allowedHeaders map[string]bool
```

Comment:
This move is to make the godoc look better. If this is placed here then there will be an empty line in the godoc between AllowedOrigins and ExposedHeaders. Like this:
```go
// AllowedOrigins determines which origins should be allowed in the
// Access-Control-Allow-Origin header.
AllowedOrigins map[string]bool

// ExposedHeaders ...
```
But if this is moved to be below all of the exported fields then the empty line is removed.